dbms/src/WindowFunctions/tests/gtest_lead_lag.cpp

```diff
@@ -36,10 +36,13 @@ class LeadLag : public DB::tests::ExecutorTest
     void executeWithConcurrencyAndBlockSize(const std::shared_ptr<tipb::DAGRequest> & request, const ColumnsWithTypeAndName & expect_columns)
     {
         WRAP_FOR_DIS_ENABLE_PLANNER_BEGIN
-        std::vector<size_t> block_sizes{1, 2, 3, DEFAULT_BLOCK_SIZE};
+        auto to_field = [](size_t block_size) {
```

Comment:
seems ugly……

Reply:
> seems ugly……

leave it as todo, since we will implement a new wf test utils.

Reply:
> > seems ugly……
> 
> leave it as todo, since we will implement a new wf test utils.

Ops, I thought you were comment on std::vector<size_t> block_sizes{1, 2, 3, DEFAULT_BLOCK_SIZE};.

I have fixed it.